pCloudServices/Set-DomainUser
# V1.3

Changes:
- Tests credentials and configuration of PSM users by default
  - Disable with -SkipPSMUserTests
- Move backups to a timestamped directory to keep Hardening folder cleaner
- Don't assume WinDomain platform numeric ID
- Check usernames not more than 20 characters and in expected format
- Add space as valid character in password
- Move reboot task to the bottom of task list
- updated some parameter names; added aliases for the previous values
